test/test.jl

```diff
@@ -1,16 +1,28 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-using Base.Test
```

Comment:
aren't we trying to move towards things being more modular and having fewer default exports?